CRVResponse/fcl/prolog_v08.fcl

```diff
@@ -11,12 +11,16 @@ BEGIN_PROLOG
     {
       module_type               : CrvStepsFromStepPointMCs
       removeNeutralParticles    : true
+      debugLevel                : 0
+      diagLevel                 : 0
+      stepPointsInstance        : "CRV"
+      stepPointsModuleLabels    : [] //This needs to be provided by the script. Use "*" for all module labels
     }
     CrvPhotons:
     {
       module_type               : CrvPhotonGenerator
-      g4ModuleLabels            : ["*"]
-      processNames              : ["*"]
+      crvStepModuleLabels       : ["*"]
+      crvStepProcessNames       : ["*"]
```

Comment:
An explicit list gives more control than "*".

Reply:
This is the prolog definition, where it is not known what labels/names are used by the actual implementation of the CrvPhotonGenerator. So I wanted to allow all module labels and process names. As an alternative, I could provide an empty list which would force the user fcl file to explicitly state which labels and process names should be used. That's what I now do for the CrvSteps module (pushed just before you wrote this comment).

Reply:
The recommended way to force the end user to supply values is:

       crvStepProcessNames : @nil

( that last token is at nil - hopefully the autocorrect won't mangle it ).  If the user fails to redefine this parameter then the code that reads it will always throw an exception at read-time.    I prefer that behaviour to having a different error downstream.

Reply:
Ok. Thank you for this recommendation.

Reply:
This may require updating once we work out the production workflow, but that's outside the scope of this PR.